algorithms/Python/arrays/equalize_the_array.py

```diff
@@ -0,0 +1,47 @@
+# Given an array of integers, determine the minimum number of elements to delete to leave only elements of equal value.
+# Example
+# arr=[1,2,2,3]
+# Delete the  2 elements 1 and 3 leaving arr=[2,2]. If both twos plus either the 1 or the 3 are deleted, it takes 3 deletions to leave either [3] or [1]. The minimum number of deletions is 2.
+
+#!/bin/python3
+
+import math
+import os
+import random
+import re
+import sys

```

Comment:
@Tanmoy07tech You can remove these imports since these are not used in the code.

Reply:
P.S. - Also add this file in [README.md](https://github.com/MakeContributions/DSA/blob/main/algorithms/Python/README.md)